tests/gireg.lua

```diff
@@ -16,6 +19,9 @@ local check = testsuite.check
 local checkv = testsuite.checkv
 local gireg = testsuite.group.new('gireg')
 
+-- Do we have Lua-side support for integers?
+local nativeIntegers = (_VERSION == 'Lua 5.3' or _VERSION == 'Lua 5.4')
```

Comment:
`_VERSION >= 'Lua 5.3`

Reply:
This seems a bit complex as `_VERSION` is a string, but probably then

```Lua
local nativeIntegers = math.maxinteger
```

as we really care about integers support and not Lua version?